lib/appium.js

```diff
@@ -27,6 +27,8 @@ class AppiumDriver extends BaseDriver {
     this.args = args;
 
     this.sessions = {};
+
+    this.pendingDrivers = {};
```

Comment:
can we standardize on `sessions` rather than `drivers`? unless we really mean a singleton driver class rather than a driver instance (corresponding to a session).

willing to discuss other options too.